egs/swbd/s5c/local/nnet3/run_raw_tdnn.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
I understand, but this is meant to be an example for using the configs with the train script. I do not have a complete example yet as the SNR predictor or SAD requires more codes and scripts to be added.